ReactWindows/ReactNative.Tests/Internal/MockJavaScriptExecutor.cs

```diff
@@ -1,56 +0,0 @@
-﻿using Newtonsoft.Json.Linq;
-using ReactNative.Bridge;
-using System;
-
-namespace ReactNative.Tests
-{
-    class MockJavaScriptExecutor : IJavaScriptExecutor
```

Comment:
MockJavaScriptExecutor comes in from Shared.Tests now. We had made a copy of it in Shared.Tests in an earlier PR leading up to this. A few other fixtures use that static mock as well.